examples/cython/sycl_buffer/use_sycl_buffer.cpp

```diff
@@ -28,7 +28,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "use_sycl_buffer.h"
-#include "dpctl_sycl_types.h"
+#include "syclinterface.h"
```

Comment:
should the header be called `dpctl_sycl_interface.h`?

Reply:
Ok, will make the change.

Reply:
done